<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>350</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Модель</string>
    </property>
    <widget class="QRadioButton" name="comp_radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Компания</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="emp_radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>50</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Сотрудник</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Действие</string>
    </property>
    <widget class="QRadioButton" name="add_radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="del_radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="insert_radioButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>350</y>
      <width>91</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Пуск</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>221</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Пользователь</string>
    </property>
    <widget class="QLineEdit" name="login_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string>Логин: </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>62</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Пароль:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwd_lineEdit">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhLatinOnly|Qt::ImhNoAutoUppercase</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>211</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Данные</string>
    </property>
    <widget class="QLineEdit" name="name_lineEdit">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>22</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Название:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>54</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Адрес:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>ИНН:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>114</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>email:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>142</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тел. номер:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="adress_lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inn_lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="email_lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="phone_lineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
